### PR TITLE
UI

### PR DESCRIPTION
**Pull request by WlodekLipski**

### CODE REVIEW

File: Assets/src/Controls/Detection/TouchDetector.cs [hunk 1]
```diff
@@ -201,6 +201,8 @@ private bool TryToResolvePlaneHold()
         {
             logger.Log("TryToResolvePlaneHold", "hold resolved");
             // at the moment ignore plane hold
```
Comment (szelinskip) [inline Assets/src/Controls/Detection/TouchDetector.cs]: This comment is no longer true and can be removed

**Review by szelinskip — COMMENTED** [review]
Please squash those commits into one and rebase ui branch onto main branch, so there is only one commit in merge request, currently it is a bit messy in git log: ui -> main merge, then main -> ui merge

**Comment by WlodekLipski** [comment]
@krecik1219 sorry for the messing up with commits.

**Comment by WlodekLipski** [comment]
[Add more materials (textures)](https://trello.com/c/f1nLEyF0/22-add-more-materials-textures)

**Comment by WlodekLipski** [comment]
[Improve UI](https://trello.com/c/l0faIIQX/21-improve-ui)

**Review by szelinskip — APPROVED** [review]

**Pull request merged**